Lib/gftools/builder/_ninja.py

```diff
@@ -134,7 +134,7 @@ def setup_rules(self):
         self.w.comment("Run the ttfautohint in-place and touch a stamp file")
         self.w.rule(
             "autohint",
-            "ttfautohint $in $in.autohinted && mv $in.autohinted $in && touch $in.autohintstamp",
+            "gftools-autohint $in && touch $in.autohintstamp",
```

Comment:
Both should be installed. In pyproject.toml we install a whole bunch of CLI utilities. I feel that using the installed name `gftools-...-...` is more robust because `gftools foo ...` can cause broken assumptions about what's in `sys.argv`.